integration-cli/docker_api_inspect_unix_test.go

```diff
@@ -19,20 +20,17 @@ func (s *DockerSuite) TestInspectApiCpusetInConfigPre120(c *check.C) {
 	dockerCmd(c, "run", "--name", name, "--cpuset", "0-1", "busybox", "true")
 
 	status, body, err := sockRequest("GET", fmt.Sprintf("/v1.19/containers/%s/json", name), nil)
-	c.Assert(status, check.Equals, http.StatusOK)
-	c.Assert(err, check.IsNil)
+	c.Assert(status, checker.Equals, http.StatusOK)
+	c.Assert(err, checker.IsNil)
 
 	var inspectJSON map[string]interface{}
-	if err = json.Unmarshal(body, &inspectJSON); err != nil {
-		c.Fatalf("unable to unmarshal body for version 1.19: %v", err)
-	}
+	c.Assert(json.Unmarshal(body, &inspectJSON),checker.IsNil,Commentf("unable to unmarshal body for version 1.19: %v",json.Unmarshal(body, &inspectJSON)))
```

Comment:
How come this is calling Unmarshal a 2nd time?
If it's already error'd out from unmarshalling, it probably won't work a 2nd time like this.